cinema/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Movie
+        fields = "__all__"
+
+    def create(self, validated_data):
+        return Movie.object.create(validated_data)
+
+    def update(self, instance, validated_data):
+        instance.description = validated_data.get("description", instance.description)
+        instance.duration = validated_data.get("duration", instance.duration)
+        instance.save()
+        return instance
```

Comment:
why do u need this here ? it seems to me there is no need to overwrite this methods if u are using modelserializer